modules/tetris/Makefile

```diff
@@ -0,0 +1,7 @@
+# Force format
+$(info $(shell astyle --style=allman tetris.c ptetris.h))
```

Comment:
This looks like it adds a dependency to the ML build system for "astyle", since all modules are built for all cams.  I don't really want to do that.  If you're volunteering to convert all ML code to be compatible, and *then* add an astyle dep, that's different!  But it also feels like a separate task.

Probably remove this dep?

Reply:
I can comment it out.

Reply:
Thanks.  Annoyingly, there's an "official" coding style in doc/CODING_STYLE - which even suggests using astyle!  But it is very clear this is not followed consistently.

Reply:
Would be nice if the entire repo was formatted consistently, but that would be too big of a diff.